eventstore/test_views.py

```diff
@@ -595,6 +596,56 @@ def test_prebirth_other_passport_origin(self):
         self.assertEqual(registration.passport_country, "other")
 
 
+class PMTCTRegistrationViewSetTests(APITestCase, BaseEventTestCase):
+    url = reverse("pmtctregistration-list")
+
+    def test_data_validation(self):
+        """
+        The supplied data must be validated, and any errors returned
+        """
+        user = get_user_model().objects.create_user("test")
+        user.user_permissions.add(
+            Permission.objects.get(codename="add_pmtctregistration")
+        )
+        self.client.force_authenticate(user)
+        response = self.client.post(self.url)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_successful_pmtct_registration_request(self):
+        """
+        Should create a new PMTCTRegistration object in the database
+        """
+        user = get_user_model().objects.create_user("test")
+        user.user_permissions.add(
+            Permission.objects.get(codename="add_pmtctregistration")
+        )
+        self.client.force_authenticate(user)
+        response = self.client.post(
+            self.url,
+            {
+                "contact_id": "9e12d04c-af25-40b6-aa4f-57c72e8e3f91",
+                "device_contact_id": "d80d51cb-8a95-4588-ac74-250d739edef8",
+                "pmtct_risk": "normal",
+                "date_of_birth": "1990-02-03",
```

Comment:
We should also have a test to confirm that `date_of_birth` can be `null` and the request will be successful, since it's not a required field.